sqlglot/parser.py

```diff
@@ -5550,10 +5578,17 @@ def _match_texts(self, texts, advance=True):
             return True
         return False
 
-    def _match_text_seq(self, *texts, advance=True):
+    def _match_text_seq(self, *texts, advance=True, upper_case=True):
```

Comment:
i don't think we need this upper_case flag, it's a bit edge casey, should be fine to just not have this check